cmd/neofs-node/config/morph/config.go

```diff
@@ -16,47 +15,36 @@ const (
 	// DialTimeoutDefault is a default dial timeout of morph chain client connection.
 	DialTimeoutDefault = 5 * time.Second
 
-	// PriorityDefault is a default endpoint priority for the morph client.
-	PriorityDefault = 1
-
 	// CacheTTLDefault is a default value for cached values TTL.
 	// It is 0, because actual default depends on block time.
 	CacheTTLDefault = time.Duration(0)
-
-	// SwitchIntervalDefault is a default Neo RPCs switch interval.
-	SwitchIntervalDefault = 2 * time.Minute
 )
 
 // RPCEndpoint returns list of the values of "rpc_endpoint" config parameter
 // from "morph" section.
 //
 // Throws panic if list is empty.
-func RPCEndpoint(c *config.Config) []client.Endpoint {
-	var es []client.Endpoint
+func RPCEndpoint(c *config.Config) []string {
+	var endpointsDeprecated []string
 
-	sub := c.Sub(subsection).Sub("rpc_endpoint")
+	sub := c.Sub(subsection).Sub("endpoint")
```

Comment:
But how is it compatible with the old config? I think you have to keep `rpc_endpoint` here the way it was. Of course it'll be deleted afterwards, but for now it should be 1:1 as in the previous config version.

Reply:
my bad, i had ~3 ideas about how that PR should look and that is how multi ideas affected it. fixed